File: game-server/src/main/java/org/forfun/mmorpg/game/player/message/ReqSelectPlayer.java
package org.forfun.mmorpg.game.player.message;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import jforgame.socket.share.annotation.MessageMeta;
import jforgame.socket.share.message.Message;
import org.forfun.mmorpg.game.player.service.PlayerService;

@MessageMeta(cmd = PlayerService.CMD_REQ_SELECT_PLAYER)
public class ReqSelectPlayer implements Message {

	@Protobuf
	private long playerId;

	public long getPlayerId() {
		return playerId;
	}

	public void setPlayerId(long playerId) {
		this.playerId = playerId;
	}
}

File: game-server/src/main/kotlin/org/forfun/mmorpg/game/player/message/ReqSelectPlayer.kt
package org.forfun.mmorpg.game.player.message

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf
import jforgame.socket.share.annotation.MessageMeta
import jforgame.socket.share.message.Message
import org.forfun.mmorpg.game.player.service.PlayerService

@MessageMeta(cmd = PlayerService.CMD_REQ_SELECT_PLAYER)
data class ReqSelectPlayer(
    @Protobuf var playerId: Long = 0
) : Message
